Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "nomad_external_dns"
+version = "0.1.0"
+edition = "2021"
+
+
+[dependencies]
+anyhow = "1.0"
+clap = { version = "4", features = ["derive"] }
+reqwest = { version = "0.12", features = ["json"] }
+serde = { version = "1.0", features = ["derive"] }
+tokio = { version = "1", features = ["full"] }
```

Comment:
I don't think you need `full` here. Can you figure out what the minimal feature set is?